images/builder/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+- test-infra-oncall
+approvers:
+- test-infra-oncall
```

Comment:
oh also clarified that test-infra-oncall should own this for now, as this tooling is expected to be trusted in the trusted cluster